diff --git a/react-native-gradle-plugin/src/main/kotlin/com/facebook/react/utils/AgpConfiguratorUtils.kt b/react-native-gradle-plugin/src/main/kotlin/com/facebook/react/utils/AgpConfiguratorUtils.kt
index c11d0ccf163224ee2743f1008aff8893ede5c191..84a3c3f17f0392c6fe0bac1d060a6f78cd4e39a9 100644
--- a/react-native-gradle-plugin/src/main/kotlin/com/facebook/react/utils/AgpConfiguratorUtils.kt
+++ b/react-native-gradle-plugin/src/main/kotlin/com/facebook/react/utils/AgpConfiguratorUtils.kt
@@ -99,7 +99,7 @@ internal object AgpConfiguratorUtils {
               .getByType(ApplicationAndroidComponentsExtension::class.java)
               .finalizeDsl { ext ->
                 ext.defaultConfig.resValue(
-                    "string", "react_native_dev_server_ip", getHostIpAddress())
+                    "string", "react_native_dev_server_ip", "127.0.0.1")
                 ext.defaultConfig.resValue("integer", "react_native_dev_server_port", devServerPort)
               }
         }
